### DISCUSSION TITLE
Comportamento estranho do po-combo

### DISCUSSION THREAD

**Discussion by viniciusgati** [po-ui/po-angular#1720] [Tire suas dúvidas] [answered]
Bom dia, tenho alguns po-combo e eles estão com um comportamento estranho, quando tenho um valor selecionado ao abrir pela primeira vez ele mostra somente o valor selecionado, clicando no mesmo e abrindo novamente ele abre o combo corretamente.

É algo que estou fazendo errado?

**Comment by CSimoesJr** [comment, marked as answer]
Boa tarde @viniciusgati, tudo bem? Estamos cientes desse comportamento e em breve analisaremos sobre uma solução.
Como medida provisória, segue exemplo do stackblitz, no qual criei uma função de teste que é chamada no html, tanto no (click), quanto no (p-input-change), para aparecer todos os valores e não apenas o selecionado:
https://stackblitz.com/edit/po-combo-default-value-error-dts39x?file=src%2Fapp%2Fapp.component.ts,src%2Fapp%2Fapp.component.html

Qualquer dúvida estou à disposição!

**Reply by viniciusgati** [reply]
ok obrigado
